gnovm/stdlibs/std/native_test.go

```diff
@@ -192,3 +196,31 @@ func TestPrevRealmIsOrigin(t *testing.T) {
 		})
 	}
 }
+
+func TestVerify(t *testing.T) {
+	kb := keys.NewInMemory()
+	pass := "hardPass"
+	info, err := kb.CreateAccount("user", DefaultAccount_Seed, pass, pass, 0, 0)
```

Comment:
You don't need to use the keybase to generate a key, you can just use something like `ed25519.GenPrivKey()` or even better `secp256k1.GenPrivKey()`, which will mimic what the keybase is doing under the hood

Not to mention it is exponentially faster 🚀